helm/charts/hpe-csi-driver/templates/nimble-csp.yaml

```diff
@@ -61,4 +62,4 @@ spec:
           key: node.kubernetes.io/unreachable
           operator: Exists
           tolerationSeconds: 30
-
+{{- end }}
```

Comment:
Insert line breaks. The linting breaks otherwise.

Reply:
will fix